src/Tickets/Commerce/Gateways/Stripe/Client.php

```diff
@@ -82,15 +82,37 @@ public function get_api_url( $endpoint, array $query_args = [] ) {
 		return add_query_arg( $query_args, "{$base_url}/{$endpoint}" );
 	}
 
+	/**
+	 * Updates the account on Stripe.
+	 *
+	 * @since TBD
+	 *
+	 * @param string $account_id
+	 * @param array  $data Which data we are using ot update the account.
+	 *
+	 * @return array|array[]|\WP_Error|null
+	 */
+	public function update_account( string $account_id, array $data = [] ) {
```

Comment:
This should be moved to the Merchant class and use the new Requests class introduced in #2299